drizzlepac/hlautils/astrometric_utils.py

```diff
@@ -615,36 +645,40 @@ def extract_sources(img, dqmask=None, fwhm=3.0, kernel=None, photmode=None,
     # convert segm to mask for daofind
     if centering_mode == 'starfind':
         src_table = None
+
         # daofind = IRAFStarFinder(fwhm=fwhm, threshold=5.*bkg.background_rms_median)
-        log.info("Setting up DAOStarFinder with: \n    fwhm={}  threshold={}".format(fwhm, dao_threshold))
-        daofind = DAOStarFinder(fwhm=fwhm, threshold=dao_threshold)
+        log.debug("Setting up DAOStarFinder with: \n    fwhm={}  threshold={}".format(fwhm, dao_threshold))
+        # daofind = DAOStarFinder(fwhm=fwhm, threshold=dao_threshold)
```

Comment:
There is a debug message here, but the line of code(Line 651)  is actually commented out.  I do see this computation at Line 680.  Should the debug statement be moved down to Line 680?

Reply:
Corrected in 219bfa6.